Content.Shared/Security/SecurityStatus.cs

```diff
@@ -1,4 +1,4 @@
-﻿namespace Content.Shared.Security;
+namespace Content.Shared.Security;
```

Comment:
sus

Reply:
yeah, to be honest, I don't know how that snuck in, I don't remember touching it. It may have been during the push/pull trying to resolve the weird conflicts between local and remote I was getting? 

I'd fix this, I just... don't really see what to do with it since, y'know, it's identical. (Also, it'll have to wait anyway, my ISP is having a meltdown rn and I don't want to try to push code with that going on.)

Reply:
Alright, ISP no longer melting down, looking at this in the IDE, and... yeah. If I delete and rewrite that it's just going to show up as another, different edit, so that's not really a solution. Is there something incredibly obvious that I, as a total newbie to coding/git/all of this, am missing?

Reply:
its a byte order mark or something

Reply:
Okay, so, today I learned what a byte order mark is. Sort of. I still have no idea how to fix this. Deleting and reentering the line didn't generate a diff.

Reply:
upstreams file has no BOM so configure your editor to not save them

if in doubt use a hex editor :trollface:

Reply:
Alright, I did a save-as UTF-8 without BOM, and changed some wording so TortoiseGit would accept that something had changed and let me push the code. Hopefully that did the trick.

Reply:
ok i had it the other way around, its upstream that has the bom not you :trollface:

Reply:
Okay, so I tested the original file and it's apparently UTF-8. But that's what I saved mine as in the commit I just pushed. 

I double checked and they have the same encoding. I am so confused.